$NetBSD: patch-rebar.config,v 1.1 2026/02/07 01:00:44 gdt Exp $

rebar objects that 27 is not 18, so just change the requirement, which
is obviously meant ot be >= 18.

--- rebar.config.orig	2026-02-06 17:09:57.758080706 +0000
+++ rebar.config
@@ -1,6 +1,6 @@
 %% -*- mode: erlang -*-
 
-{require_otp_vsn, "18"}.
+{require_otp_vsn, "27"}.
 
 {erl_opts, [debug_info]}.
 {yrl_opts, [{includefile, "include/erlydtl_preparser.hrl"}]}.
